internal/batchpin/batchpin.go

```diff
@@ -32,42 +31,33 @@ type Submitter interface {
 
 type batchPinSubmitter struct {
 	database   database.Plugin
-	identity   identity.Plugin
+	identity   identity.Manager
 	blockchain blockchain.Plugin
 }
 
-func NewBatchPinSubmitter(di database.Plugin, ii identity.Plugin, bi blockchain.Plugin) Submitter {
+func NewBatchPinSubmitter(di database.Plugin, im identity.Manager, bi blockchain.Plugin) Submitter {
 	return &batchPinSubmitter{
 		database:   di,
-		identity:   ii,
+		identity:   im,
 		blockchain: bi,
 	}
 }
 
 func (bp *batchPinSubmitter) SubmitPinnedBatch(ctx context.Context, batch *fftypes.Batch, contexts []*fftypes.Bytes32) error {
 
-	signingIdentity, err := bp.identity.Resolve(ctx, batch.Author)
-	if err == nil {
-		err = bp.blockchain.VerifyIdentitySyntax(ctx, signingIdentity)
-	}
-	if err != nil {
-		log.L(ctx).Errorf("Invalid signing identity '%s': %s", batch.Author, err)
-		return err
-	}
-
```

Comment:
I guess it's now assumed that the identity has been pre-verified before invoking this helper?

Reply:
Yes. It comes from the messages in the batch which must each have been resolved, and will be verified by all members processing the batch, so doing an additional pre-submission check here does not seem efficient.